### PR TITLE
CMS-1745 invest redirects

### PR DESCRIPTION
**Pull request by sdonk**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: invest/helpers.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+LANGUAGE_CODES = ('es', 'de', 'fr', 'pt', 'zh-hans', 'ar', 'ja')
```
Comment (eadpearce) [inline invest/helpers.py]: these should match our list of currently supported languages so if the code isn't in here it defaults to english right?

Comment (sdonk) [reply invest/helpers.py]: yes

Comment (eadpearce) [reply invest/helpers.py]: then why is `ar` still in here? 🤣

Comment (sdonk) [reply invest/helpers.py]: To allow this https://github.com/uktrade/great-international-ui/pull/261/files#diff-ee6762a72813838213b6eb58760f4994R72

Comment (eadpearce) [reply invest/helpers.py]: oh okay. i thought we were still supporting japanese?

Comment (sdonk) [reply invest/helpers.py]: The spreadsheet says ar and ja should go to English

**Pull request merged**